zerver/lib/bugdown/__init__.py

```diff
@@ -568,6 +613,20 @@ def vimeo_image(self, url: Text) -> Optional[Text]:
             return "http://i.vimeocdn.com/video/%s.jpg" % (vm_id,)
         return None
 
+    def message_quote_id(self, url: Text) -> Optional[Text]:
```

Comment:
How about renaming this function to `get_message_id` to present its function better?